src/app/modules/shared-components/components/loading/loading.component.ts

```diff
@@ -0,0 +1,25 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'alg-loading',
+  templateUrl: './loading.component.html',
+  styleUrls: [ './loading.component.scss' ],
+})
+export class LoadingComponent {
+  @Input() size: 'small' | 'medium' | 'large' = 'large';
+
+  styles = {
+    'small': {
+      width: '24px',
+      height: '24px',
```

Comment:
I think we should use the equivalent rem values for consistency

Reply:
Fixed